checkpointctl.go

```diff
@@ -84,83 +114,110 @@ func setupShow() *cobra.Command {
 		false,
 		"Display all additional information about the checkpoints",
 	)
+	flags.StringVar(
+		&format,
+		"format",
+		"tree",
+		"Format the output using a tree or json format",
```

Comment:
I think something like this would be better in case we decide to support some other serialisable format later on.
```suggestion
		"Specify the output format [tree (default), json]",
```